src/index.js

```diff
@@ -49,8 +49,8 @@ const about = {
 	serviceTier: 'bronze',
 	_hostname: process.env.HEROKU_DYNO_ID || os.hostname(),
 	contacts: [{
-		name: 'Matt Brennan',
-		email: 'matthew.brennan@ft.com',
+		name: 'Kara Brightwell',
+		email: 'kara.brightwell@ft.com',
```

Comment:
I think it would be better if this went to the team rather than a single person who no longer works on the apps.

I am not 100% what uses these `/__about` endpoints so I am making the assumption the value of `name` is just a free text field

```suggestion
		name: 'Customer Products App team',
		email: 'ftmobile@ft.com',
```